packages/dnb-eufemia/src/shared/locales/sv-SE.ts

```diff
@@ -0,0 +1,168 @@
+import nb from './nb-NO'
+export default {
+  'sv-SE': {
+    TextCounter: {
+      characterDown: '%count av %max tecken återstår.',
+      characterUp: 'Du har använt %count av %max tecken.',
+      characterExceeded: '%count tecken över gränsen på %max.',
+    },
+    TimelineItem: {
+      alt_label_completed: 'Utförd',
+      alt_label_current: 'Nuvarande',
+      alt_label_upcoming: 'Kommande',
+    },
+    Breadcrumb: {
+      navText: 'Sidhierarki',
+      goBackText: 'Tillbaka',
+      homeText: 'Hem',
+      backToText: 'Tillbaka till...',
+    },
+    DatePicker: {
+      day: 'dag',
+      month: 'månad',
+      year: 'år',
+      start: 'från',
+      end: 'till',
+      selectedDate: 'Vald datum: %s',
+      selectedMonth: 'Vald månad %s',
+      selectedYear: 'Vald år %s',
+      nextMonth: 'Nästa månad %s',
+      prevMonth: 'Föregående månad %s',
+      nextYear: 'Nästa år %s',
+      prevYear: 'Föregående år %s',
+      openPickerText: 'öppna datumväljare',
+      maskOrder: 'dd/mm/yyyy',
+      maskPlaceholder: 'dd.mm.åååå',
+      dateFormat: 'yyyy-MM-dd',
+      returnFormat: 'yyyy-MM-dd',
+      submitButtonText: 'Ok',
```

Comment:
```suggestion
      submitButtonText: 'Okej',
```

I think this is more Swedish, at least I see this in a couple of Swedish sites I've browsed through.